conf/chi.conf.example

```diff
@@ -17,7 +17,7 @@ expires_in=1d
 expires_on_backend=1
 
 [storage httpd.portal]
-expires_in=10m
+expires_in=1d
```

Comment:
10 minutes is too small however I think 1 day is too much.
I think one hour is a good place to start.